hack/whitespace.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+#
+# Check for trailing whitespaces in all tracked files.
+#
+# Usage:
+# hack/whitespace.sh # Check for trailing whitespaces
+# hack/whitespace.sh --fix # Drop trailing whitespaces
+
+set -e
+
+function fix() {
+    git ls-files -- ':!vendor/' | xargs sed --follow-symlinks -i 's/[[:space:]]*$//'
```

Comment:
can we put in a function `git ls-files -- ':!vendor/'` command we are using it twice.

Reply:
The rest of the line is different, I would rather keep it than have a function which would have a name practically saying the same.